app/controllers/api/v2/courses_controller.rb

```diff
@@ -60,8 +60,10 @@ def publishable
       def update
         update_enrichment
         update_sites
+        update_gcse_requirements
```

Comment:
Does this need to be a separate method? Arent we just updating the course object itself?

Reply:
Just discovered deserialisers 🤦‍♂. Will do this. It's way easier 👍